openassessment/assessment/api/peer.py

```diff
@@ -244,9 +249,19 @@ def get_score(submission_uuid, peer_requirements, course_settings):
 
     # Check if enough peers have graded this submission
     num_required_peer_grades = required_peer_grades(submission_uuid, peer_requirements, course_settings)
-    if items.count() < num_required_peer_grades:
+    num_recieved_peer_grades = items.count()
+    if num_recieved_peer_grades < num_required_peer_grades:
```

Comment:
Nit: the order / comments of these is confusing. It isn't clear to me on first read that `num_required_peer_grades` is not the actual configured amount but can morph if flex grading is active. Maybe putting the flex grading check above this or extending the comment to that effect would be helpful.